apps/src/code-studio/pd/application/facilitator1819/Facilitator1819Labels.jsx

```diff
@@ -19,7 +19,8 @@ const pageLabels = {
     institutionType: "What type of institution do you work for?",
     currentEmployer: "Current employer",
     jobTitle: "What is your job title?",
-    resumeLink: "Please provide a link to your resume, LinkedIn profile, website, or summarize your relevant past experience.",
+    resumeLink:
```

Comment:
Not now - but we need to fix this later. We should ask for resume uploads I think